portfolio.css

```diff
@@ -0,0 +1,899 @@
+  position: relative;
+  margin-right: 0;
+}
+
+.section2 {
+  position: relative;
+  width: 375px;
+  background: #fff;
+  top: 0;
+  left: 0;
+  height: 3000px;
+  overflow-x: hidden;
+}
+
+.myRecent {
+  display: grid;
+  grid-template-columns: auto;
+  overflow-x: hidden;
+}
+
+.recent {
+  position: absolute;
+  width: 327px;
+  height: 83px;
+  margin-left: 12px;
+  top: 0;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding: 24px;
+}
+
+.vector1 {
+  position: absolute;
+  width: 310px;
+  height: 0;
+  margin-left: 24px;
+  top: 107px;
+  border: 1px solid #212121;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-top: 24px;
+  align-items: center;
+  overflow-x: hidden;
+}
+
+.image1 {
+  position: absolute;
+  max-width: 327px;
+  height: 237px;
+  left: 24px;
+  top: 200px;
+  overflow-x: hidden;
+}
+
+.multi {
+  position: absolute;
+  width: 327px;
+  height: 36px;
+  left: 0;
+  top: 440px;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 24px;
+}
+
+.selection {
+  position: static;
+  width: 327px;
+  height: 147px;
+  margin-left: 24px;
+  margin-top: 520px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.buttons {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  position: relative;
+  width: 100px;
+  height: 40px;
+  margin-top: -40px;
+  margin-left: -30px;
+}
+
+.btn {
+  padding: 8px;
+  margin: 8px;
+}
+
+.btn:hover {
+  border: 2px solid blue;
+  text-decoration: underline;
+}
+
+.see {
+  position: relative;
+  max-width: 100px;
+  max-height: 24px;
+  left: 18px;
+  top: 12px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  background: #ff6b00;
+  padding: 8px;
+}
+
+.see:hover {
+  border: 2px solid blue;
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
+.see1 {
+  font-weight: 700;
+  font-size: 15px;
+  line-height: 24px;
+  padding: 2px;
+  position: relative;
+  left: -8px;
+  color: #fff;
+  background: #ff6b00;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  text-decoration: none;
+  margin-left: 8px;
+}
+
+.see1:hover {
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
+.profesional {
+  width: 375px;
+  display: grid;
+  grid-template-columns: auto;
+  background-image: url(../portfolio-mobile-version/images/Placeholder12.png);
+  overflow-x: hidden;
```

Comment:
## [Optional]
### Please make these changes :+1:
![re-review-last](https://user-images.githubusercontent.com/93039359/176759090-403abad9-5b06-4472-9639-20b515bf2e0e.png)
**[Optional]**
- [ ] Please make your link something like the above in the picture. Then your project will show all the images because you are not linking properly.